docs/sphinx/manual/hpx_runtime_and_resources.rst

```diff
@@ -216,6 +237,94 @@ pool. Below is an example of adding all processing units from the first NUMA
 domain to a custom thread pool, unless there is only one NUMA domain in which
 case we leave the first processing unit for the default thread pool:
 
+.. code-block:: c++
```

Comment:
This should also be in a file if possible, and if needed. I'd say this isn't needed. The only thing you'd need to show here is the change from using the resource partitioner inline before `hpx::init` to wrapping the calls to the rp in a lambda.

Reply:
So to be sure I understand what you want, you just want a code block of like 2 lines to show the old way and another one with the callback to show the new way without any includes or else?